src/pages/Facility.tsx

```diff
@@ -1,18 +1,43 @@
 import MainLayout from 'src/layouts/main';
 import { FacilityIntroduction } from '../containers/facility/FacilityIntroduction';
 import { FacilityOffers } from 'src/containers/facility/FacilityOffers';
+import { Breadcrumbs } from '../components/Breadcrumbs';
+import { createSearchParams } from 'react-router-dom';
+import { useAccommodationsAndOffers } from '../hooks/useAccommodationsAndOffers.tsx';
+import { useMemo } from 'react';
 
 export const Facility = () => {
+  const { latestQueryParams } = useAccommodationsAndOffers();
+
+  const query = useMemo(() => {
+    if (latestQueryParams === undefined) {
+      return '';
+    }
+
+    const params = {
+      roomCount: latestQueryParams.roomCount.toString(),
+      adultCount: latestQueryParams.adultCount.toString(),
+      arrival: latestQueryParams.arrival?.toDateString() ?? '',
+      departure: latestQueryParams.departure?.toDateString() ?? '',
+      location: latestQueryParams.location
+    };
+    return createSearchParams(params);
+  }, [latestQueryParams, createSearchParams]);
+
   return (
-    <MainLayout
-    //TODO: check with designer whether breadcrumbs are still needed.
-    // breadcrumbs={[
-    //   {
-    //     label: 'Search',
-    //     path: `/`
-    //   }
-    // ]}
-    >
+    <MainLayout>
+      <Breadcrumbs
+        links={[
+          {
+            name: 'Home',
+            href: '/'
+          },
+          {
+            name: 'Search',
+            href: `/search?${query}`
```

Comment:
Have you tested whether this is working?

Reply:
yup, as expected, no additional fetch, and facility list still there. new hook works amazing